src/screens/KeyManagement/OCR2KeysCreate.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import TextField from '@material-ui/core/TextField'
+import MenuItem from '@material-ui/core/MenuItem'
+import { ConfirmationDialog } from 'components/Dialogs/ConfirmationDialog'
+import { useOCR2KeyFamilyQuery } from 'hooks/queries/useOCR2KeysQuery'
+
+export interface Props {
+  showCreateKeyDialog: boolean
+  setToggleCreateKeyDialog: (toggleCreateKeyDialog: boolean) => void
+  onCreate: (chainType: string) => Promise<any>
+}
+
+export const OCR2KeysCreate: React.FC<Props> = ({
+  showCreateKeyDialog,
+  setToggleCreateKeyDialog,
+  onCreate,
+}) => {
+  const [createOCR2KeyType, setCreateOCR2KeyType] = React.useState<
+    string | undefined
+  >(undefined)
+
+  const handleChainTypeChange = (
+    event: React.ChangeEvent<{ value: unknown }>,
+  ) => {
+    setCreateOCR2KeyType(event.target.value as string)
+  }
+
+  const createOCR2KeyHandler = () => {
+    setToggleCreateKeyDialog(false)
+    if (createOCR2KeyType == undefined) {
+      return
+    }
+    onCreate(createOCR2KeyType)
+  }
+
+  const { loading, error, data } = useOCR2KeyFamilyQuery()
+  if (loading || error) return <div />
+
+  const supportedChainFamilies: Array<string> = data?.__type.enumValues.map(
+    (chainFamily: { name: any }) => {
+      return chainFamily.name
+    },
+  )
+  createOCR2KeyType == undefined &&
+    setCreateOCR2KeyType(supportedChainFamilies[0])
+
+  return (
+    <ConfirmationDialog
+      open={!!showCreateKeyDialog}
```

Comment:
```suggestion
      open={showCreateKeyDialog}
```

Shouldn't need to do a type cast here, its a boolean

Reply:
Changed in [`6bb1ac3` (#38)](https://github.com/smartcontractkit/operator-ui/pull/38/commits/6bb1ac37ef5ef0b55deb0414fff2035857755311)